vms/proposervm/client.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
I think this file is redundant. It's the same client as in `vms/proposervm/client_jsonrpc.go`